src/Sylius/Bundle/AdminBundle/Resources/private/js/semantic-ui-calendar.js

```diff
@@ -0,0 +1,11 @@
+(function($) {
+    $(document).ready(function() {
+        $('.calendar').calendar();
+        $('#rangestart').calendar({
```

Comment:
I have overridden a default datetime templates to add this calendar to them. I thought about something like `full calendar`, `date calendar` and `time calendar`(I know how it sounds like) classes to determine this UI